src/Marten/Linq/Parsing/MethodCallParser.cs

```diff
@@ -10,7 +10,7 @@ public abstract class MethodCallParser<T> : IMethodCallParser
     {
         private readonly MethodInfo _method;
 
-        public MethodCallParser(Expression<Action<T>> method)
+        protected MethodCallParser(Expression<Action<T>> method)
```

Comment:
Ugh, please definitely don't do that one. Way annoying with R# when you start implementing subclasses.